frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -649,6 +666,18 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxElectableTargets: Get<SolutionTargetIndexOf<Self>>;
 
+		/// Maximum number of backers that a single winner may have in the election.
```

Comment:
you could say "read `ElectionProvider::MaxBackerPerWinner`" instead of duplicating.